tests/test_tiingo.py

```diff
@@ -107,6 +107,21 @@ def test_intraday_ticker_price(self):
                                                frequency="30Min")
         self.assertGreater(len(prices), 1)
 
+    @vcr.use_cassette('tests/fixtures/list_stock_tickers.yaml')
+    def test_list_all_stock_tickers(self):
+        tickers = self._client.list_tickers()
+        assert len(tickers) > 1
+        assert any(ticker['assetType' == 'Stock' for ticker in tickers)
+        assert any(ticker['assetType' == 'Mutual Fund' for ticker in tickers)
+        assert any(ticker['assetType' == 'ETF' for ticker in tickers)
+
+    def test_list_multi_stock_tickers(self):
```

Comment:
I think we may want to include a cassette for this test as well.

Reply:
fixed